Dockerfile

```diff
@@ -18,6 +18,10 @@ WORKDIR /app
 COPY --from=build /app_output .
 COPY --from=build /Storage/Migration ./Migration
 
+# Add support for not only English cultures
+RUN apk add --no-cache icu-libs icu-data-full tzdata
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
```

Comment:
I'm a little worried that changing this might change behavior in unintended ways around in the Storage application.

Reply:
We'll do a before and after comparison of a few API responses.